Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM centos:7
+FROM centos:7 as cfgov-runtime
 
-# Install required packages
+# Install build-required packages
 RUN yum install -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-centos10-10-2.noarch.rpm && \
-    yum install -y epel-release make gcc gcc-c++ kernel-devel mailcap which && \
-    yum install -y postgresql10 postgresql10-devel && \
-    yum install -y python-devel python36 python36-devel && \
-    yum clean all
+    yum install -y epel-release mailcap which httpd && \
+    yum install -y postgresql10  && \
+    yum install -y python36 && \
+    yum clean all && rm -rf /var/cache/yum
+
+ENV PYTHON /usr/bin/python3
+ENV PYTHONPATH=/src/cfgov-refresh/cfgov/
+COPY docker/python/entrypoint.sh /usr/local/bin/entrypoint.sh
+RUN mkdir /active-python/ && chown -R apache:apache /active-python
+
+ENTRYPOINT ["sh", "/usr/local/bin/entrypoint.sh"]
+CMD ["sh", "-c", "/active-python/python /src/cfgov-refresh/cfgov/manage.py runmodwsgi --port 8000 --user apache --group apache --log-to-terminal --working-directory /src/cfgov-refresh/  $EXTRA_MODWSGI_ARGS"]
+EXPOSE 8000
+
+
+FROM cfgov-runtime AS cfgov-dev
+
+# Install build-required packages
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn make gcc gcc-c++ kernel-devel unzip && \
+    yum install -y httpd-devel postgresql10-devel && \
+    yum install -y python-devel python-pip python36-devel && \
+    yum clean all && rm -rf /var/cache/yum
+RUN npm install -g gulp
```

Comment:
Is global gulp required here? I believe we're intending to use the gulp local to the project

Reply:
I'm not sure-- maybe I'll take it out and see if anything breaks

Reply:
You were right, that line was unneccesary